gnc/sub8_perception/nodes/marker_occ_grid.py

```diff
@@ -295,9 +295,7 @@ def get_tf(self, timestamp=None):
         x_y_position = trans[:2]
         self.tf_listener.waitForTransform("/ground", "/downward", timestamp, rospy.Duration(1.0))
         trans, _ = self.tf_listener.lookupTransform("/ground", "/downward", timestamp)
-        height = trans[2]
-
-        #self.correct_height(height, timestamp)
+        height = np.nan_to_num(trans[2])
```

Comment:
`height` here was nan, that's what was causing the bugs. It seems like a weird bug that shouldn't happen since if the tf isn't found it is supposed to thrown an error.